frontend/src/book-read/components/reviews.component.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+import { useFetchReviewsByBookQuery } from "../store/apis/review";
+import { SHARED_CONFIG } from "../../shared";
+import { ReviewPaginatedList } from "../../shared/components/lists/reviews-paginated-list.jsx";
+
+export default function Reviews() {
+  const [currentPage, setCurrentPage] = useState(1);
+  const { bookId } = useParams();
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Eslint ignores should be removed past development.

Reply:
Removed